pom.xml

```diff
@@ -20,9 +20,9 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <build-support.version>1.5</build-support.version>
     <checkstyle.version>8.37</checkstyle.version>
-    <slf4j.version>2.0.0-alpha1</slf4j.version>
+    <slf4j.version>2.0.0-alpha4</slf4j.version>
     <log4j2.version>2.14.0</log4j2.version>
-    <logback.version>1.3.0-alpha5</logback.version>
+    <logback.version>1.3.0-alpha9</logback.version>
```

Comment:
might need change as well in `jetty-home/src/main/resources/modules/logging-logback.mod`

and same for the files with 
```
[ini]
slf4j.version?=2.0.0-alpha1
```

Reply:
Done.